provisioning/group_vars/mangaki_standalone

```diff
@@ -4,6 +4,7 @@ mangaki_source_db_user: '{{ mangaki_db_user }}'
 mangaki_source_db_name: '{{ mangaki_db_name }}'
 mangaki_source_db_password: '{{ mangaki_db_password }}'
 mangaki_source_static_root: '{{ mangaki_web_www_root }}/static'
+mangaki_source_data_root: '{{ mangaki_web_www_root }}/data'
```

Comment:
We don't want to expose our side-data to the Internet. Rather `/home/{{user}}/data` or `/opt/mangaki/`, something around these lines, sounds good.

Reply:
Ou directement /data ? Pas trop sale ?

Reply:
Trop sale

Reply:
Done.